packages/mobile/src/import/ImportWallet.tsx

```diff
@@ -97,6 +101,8 @@ export class ImportWallet extends React.Component<Props, State> {
   }
 
   onPressRestore = () => {
+    const { route, navigation } = this.props
+    const useEmptyWallet = !!route.params?.showZeroBalanceModal
```

Comment:
Why is this in the nav props instead of the redux store? Just for convenience?

Reply:
Well, it is only specific to this very screen and does not affect any business logic outside of it. Nav prop felt right for me.